docs/gambit/mutation-verifier.md

```diff
@@ -64,6 +73,21 @@ In `gambit.conf`:
 }
 ```
 
+### Manual Mutations
+You can add manual mutations to `mutation.conf` like so:
+
+```json
+{
+  "filename" : "C.sol",
+  "num-mutants": 5,
+  "manual_mutants": {
+     "C.sol": "path/to/dir/with/manual_mutants/for/C"
+  }
+}
+```
+If you set `num_mutants` to 0 in the above file, `gambit` will not generate any mutants, you will only run
```

Comment:
I think `Gambit` should be capitalized as it is a name? Is `gambit` the command that is run?

Reply:
yes it's `gambit` not `Gambit`

Reply:
i think we should use `gambit` because that is what we are running, no?